tritium-lib/src/main/java/com/palantir/tritium/Tritium.java

```diff
@@ -44,4 +44,22 @@ private Tritium() {
                 .build();
     }
 
+    /**
+     * Return an instrumented proxy of the specified service interface and delegate that records aggregated invocation
+     * metrics and performance trace logging.
+     *
+     * @param serviceInterface service interface
+     * @param delegate delegate to instrument
+     * @param metricRegistry metric registry
+     * @param globalPrefix shared prefix across service interfaces for Methods annotated with MetricGroup
+     * @return instrumented proxy implementing specified service interface
+     */
+    public static <T, U extends T> T instrument(
+            Class<T> serviceInterface, U delegate, MetricRegistry metricRegistry, String globalPrefix) {
```

Comment:
I'm leaning toward not adding this method here in `Tritium` as it is probably clearer for folks that want control over global prefix to use the `Instrumentation` builder explicitly

```
Instrumentation.builder(serviceInterface, delegate)
                .withMetrics(metricRegistry, globalPrefix)
                .withPerformanceTraceLogging()
                .build();
```

Reply:
cool, can remove that